### ISSUE TITLE
How to add custom commands

### ISSUE THREAD

**Issue by gmag11** [esphome/esp-web-tools#546]
Can we add custom commands to trigger additional improv RPC in the board to configure more settings besides WiFi credencials and show info in the flash dialog?

**Comment by yanshay** [comment]
I'm new to this but I think the way to make additional configurations is by providing a URL through Improve Wifi, so Visit Device will pop a web configuration application for additional configs.

**Comment by balloob** [comment]
Yep, @yanshay is correct. The next URL is exactly to offer your own onboarding menu.
